ivadomed/transforms.py

```diff
@@ -544,6 +546,22 @@ def __call__(self, sample, metadata=None):
             return sample, metadata
 
 
+class BoundingBoxCrop(Crop):
```

Comment:
Great!
Were you considering adding a undo_transform for testing time purposes? or maybe thats done somewhere else? or for another PR..

Reply:
The undo is accessible to its parent Crop :)!